ramls/examples/itemdamagedstatus.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Personally, I might separate the words in the file name to make it easier to read

Reply:
it's weird, I agree. But many of those files have the same style, so I decided that's a rule.

Reply:
I don't think it is a rule.

I agree that it is valuable to be consistent. Which means we either
* start a new precedent during this pull request, and then issue another soon after to rename all of the other files
* leave it as it is, including these new files, during this pull request and change to a different convention in a separate pull request
* leave it as is

What do you think?

Reply:
I would crate a separate ticket in which scope will create a pull request
(Probably add something else TODO etc.)

Reply:
Ok, we shall leave it for this pull request